src/demi-glace.md

```diff
@@ -0,0 +1,23 @@
+# Demi-glace
+This is a take on a traditional demi-glace without needing to buy an astronomically large amount of veal bones. Use this as your main base for pan sauces, an extra flavor bomb, or a replacement for recipes that call for broth or stock.
+
+These are the basic proportions — multiply as needed. I typically multiply by six to give me several weeks worth of demi-glace ice cubes.
+
+## Ingredients
+- 1 qt (32 oz, 946mL) **low-sodium** broth or stock (I prefer Pacific Foods)
+- 1 1/4 oz (7g) packet **unflavored, unsweetened** gelatin
+- 1 teaspoon (5mL) tomato paste
+- 1 teaspoon (5mL) **low-sodium** soy sauce
+- 1/2 teaspoon onion powder (_very_ optional)
+
+## Directions
+1. Put the tomato paste in a large pot, _then_ turn the heat on medium
+2. Spread the paste around the bottom of the pot and stir until it just barely starts to brown
+3. Deglaze with the broth
+4. Stir in all other ingredients and boil over high heat until reduced to about 1/6th of its original volume
+5. Freeze in ice cub trays
+
+## Contribution
+Dan - ETH `0xC86468549c180A296384aAC2F76c8F581691Bc1b`
+
+;tags: sauce basic french
```

Comment:
Might want to change your formatting. According to the official [Markdown Guide](https://www.markdownguide.org/getting-started/), you are supposed to have a space between headers and their child-text.